src/ResourceManager/Maps/Commands.Maps/AzureRM.Maps.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.0.5'
+ModuleVersion = '1.0.6'
```

Comment:
@SudhindraKovalam you can revert the change in this file -- this file is going to be deleted in favor of `Az.Maps.psd1` in the next week